lib/autobuild/import/git.rb

```diff
@@ -477,6 +523,9 @@ def self.run_git_bare(package, *args)
         #
         # Set a remote up in the repositorie's configuration
         def setup_remote(package, remote_name, repository, push_to = repository)
+            if !has_all_branches?
```

Comment:
Use `unless` rather than negative condition.

```ruby
unless has_all_branches
    relocate(repository, default_branch: default_branch(package))
end
```